### DISCUSSION TITLE
Is it possible to hide / show template card based on entity value?

### DISCUSSION THREAD

**Discussion by jpmiller25** [piitaya/lovelace-mushroom#1013] [Q&A]
I'd like to use template card as a kind of notification, up front on the dashboard if freezer temp goes high. If it's under 0, it never displays, if above it will show red and display current temp. I've made the template card but can't find any way to automate if it's visible or not. Or is there another way to go about this?

**Comment by jdiegmueller** [comment]
I will not claim this is the Best(tm) or Most RIght(tm) way to do this, but since no one has responded to your request for a handful of days I figured I would chime in with how I'm doing something similar: I'm using card_mod to selectively insert some CSS to hide the card when I don't want to see it:
```
  card_mod:
    style: |
      ha-card {
            {% if is_number(states('sensor.whatever_air_temperature')) and states('sensor.whatever_air_temperature') | float <= 0 %}
              display: none !important;
            {% endif %}
      }
```

**Reply by andynash** [reply]
I use the built-in Conditional card for this (type: conditional).

Perhaps it would be useful for the template card to include a conditional display option so more complex yaml can be used, but this seems to be enough for me so far.

**Reply by andynash** [reply]
Ps for the two use cases described above, if the conditional card isnt sufficient, you could create a template sensor/binary sensor, and use this with the conditional card.
